app/pom.xml

```diff
@@ -77,7 +77,7 @@
     <commons.compress.version>1.18</commons.compress.version>
 
     <!-- Global camel version used everywhere -->
-    <camel.version>2.21.0.fuse-740038</camel.version>
+    <camel.version>2.21.0.fuse-750007</camel.version>
```

Comment:
Check for the version. There are least two other locations to change

Reply:
Extension and Integration BOM as far I can see, thanks for pointing out!